### PR TITLE
Update github_user_access_request.md

### PR DESCRIPTION
**Pull request by patricksimonian**
Updated the github issue with a link to the just ask instance :)

### CODE REVIEW

**Review by ShellyXueHan — APPROVED** [review]
Just Ask is up! dope!

File: .github/ISSUE_TEMPLATE/github_user_access_request.md [hunk 1]
```diff
@@ -4,8 +4,18 @@ about: To grant or remove access for a team member to BCGOV GitHub repositories.
 title: ''
 labels: github-membership
 assignees: caggles, ShellyXueHan, mitovskaol, patricksimonian
-
 ---
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/github_user_access_request.md:7]:
```suggestion
---
## What we use GitHub Org Membership for

We use the membership in the GitHub org - bcgov and BCDevOps - to control access to the following functions on the DevOps Openshift Platform: 
- a membership in **bcgov** org allows teams to create **public** repos in bcgov org and to deploy their code to the Openshift Platform
- a membership in **BCDevOps** org grants developers access to the Openshift 4 Console (Silver cluster). Note that it does **not** grant access to any namespaces in Openshift. Access to the namespaces is granted to Product Owner and Technical Lead through the Project Registry at the time of the namespace creation or can be granted by them to other team members.
```

File: .github/ISSUE_TEMPLATE/github_user_access_request.md [hunk 2]
```diff
@@ -4,8 +4,18 @@ about: To grant or remove access for a team member to BCGOV GitHub repositories.
...
+## Github Access
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/github_user_access_request.md:8]:
```suggestion
## Changes to Github Organization Membership Requests
```

File: .github/ISSUE_TEMPLATE/github_user_access_request.md [hunk 3]
```diff
@@ -4,8 +4,18 @@ about: To grant or remove access for a team member to BCGOV GitHub repositories.
...
+
+There have been changes to the Github Access management workflow to __better automate__ the process. Please read the details below.
+### Gaining Access
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/github_user_access_request.md:11]:
```suggestion
### Requesting a membership in bcgov and BCDevOps orgs
```

**Review by mitovskaol — APPROVED** [review]

**Pull request merged**